backend/services/src/shared/email-helper/email-helper.service.ts

```diff
@@ -43,10 +43,16 @@ export class EmailHelperService {
 
     switch (template.id) {
       case "PROGRAMME_REJECTION":
+        let rejectedDate = new Date(programme.txTime);
```

Comment:
We have luxon library to date formatting

Reply:
fixed. installed luxon in the backend